README.md

```diff
@@ -1,25 +1,66 @@
+![Latest release](https://img.shields.io/github/release/common-workflow-language/common-workflow-language.svg?style=flat)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/common-workflow-language/common-workflow-language?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![GitHub stars](https://img.shields.io/github/stars/common-workflow-language/common-workflow-language.svg)](https://github.com/common-workflow-language/common-workflow-language/stargazers)
+
 Common Workflow Language
 ========================
 
-[**Support**](#Support) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/common-workflow-language/common-workflow-language?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![GitHub stars](https://img.shields.io/github/stars/common-workflow-language/common-workflow-language.svg)](https://github.com/common-workflow-language/common-workflow-language/stargazers)
+* Describe analysis workflows and tools of arbitrary complexity using plain text
+* Write once, test locally, then scale up and run on cluster, cloud and HPC environments
+  - No lock in: CWL is supported by a [wide variety](#Implementations) of open-source 
+    and commercial executors and platform vendors
+  - Conformance test suite and [public conformance result display][conf-test] monitors 
+    uniformity of executor behavior
+* Bake input validations into tool description, protect from costly user errors
+* Use docker images to freeze executable, allow users to reproducibly run workflows
+* Nest any tool or workflow to arbitrary depth
+  - Each CWL document can be run by itself, or as part of a larger workflow, thanks to its
+    strict interface specification
+
+[conf-test]: https://ci.commonwl.org/job/cwltool-conformance/lastBuild/version=v1.0/testReport/
```

Comment:
This should just link to the root https://ci.commonwl.org rather than cwltool specifically?

Reply:
My thinking was to show an actual conf test report and the detail which we expose to external scrutiny. I think this is really amazing and a strong point of the technical part of the spec + implementation.